lib/publiccloud/provider.pm

```diff
@@ -375,16 +375,17 @@ Depending on the method (get|post) you can pass additional data as json.
 =cut
 sub vault_api {
     my ($self, $path, %args) = @_;
-    my $method = $args{method} // 'get';
-    my $data   = $args{data}   // {};
-    my $ua     = Mojo::UserAgent->new;
-    my $url    = get_required_var('_SECRET_PUBLIC_CLOUD_REST_URL');
+    my $method    = $args{method} // 'get';
+    my $data      = $args{data}   // {};
+    my $ua        = Mojo::UserAgent->new;
+    my $url       = get_required_var('_SECRET_PUBLIC_CLOUD_REST_URL');
+    my $namespace = get_var('PUBLIC_CLOUD_VAULT_NAMESPACE');
```

Comment:
Lets give it a default of ``. 
I think we would run in an "use of uninitialized variable in string concat" in line 388.

Reply:
That true, done!